### DISCUSSION TITLE
Welcome to Open-source-Practice Discussions!

### DISCUSSION THREAD

**Discussion by sanjay-kv** [sanjay-kv/Open-source-Practice#404] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanjay-kv** [comment]
I will review the merge soon. Meanwhile all can connected over here: https://www.linkedin.com/in/sanjay-k-v/

**Reply by Chalseyji** [reply]
I'm not able to merge till now. Please grant access.
